libsolc/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+set(
```

Comment:
Not needed?

Reply:
I was wondering about this. The same applies to `solc/CMakeLists.txt` too I guess.

Reply:
The `set()` just sets the variable value. It is `sources = ""`. If `sources` are not used anywhere remove it.